packages/dd-trace/src/config.js

```diff
@@ -802,6 +627,85 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     this._setBoolean(env, 'logInjection', DD_LOGS_INJECTION)
     this._setArray(env, 'headerTags', DD_TRACE_HEADER_TAGS)
     this._setTags(env, 'tags', tags)
+    this._setBoolean(env, 'tracing', !isFalse(DD_TRACING_ENABLED))
+    this._setValue(env, 'dbmPropagationMode', DD_DBM_PROPAGATION_MODE)
```

Comment:
Strings should use `_setString_` to avoid empty strings being used as actual values.